staging/jenkins/values.yaml

```diff
@@ -0,0 +1,19 @@
+master:
+  installPlugins: ["prometheus:2.0.6"]
+  adminPassword: "deleteme"
```

Comment:
This is a good place to add a brief comment explaining how you expect the end-user to handle this password field.

For Kubeaddons one of the goals is for Addon resources to be "ready to go" without modification, while still being secure. For other services, we've used [initContainers](https://github.com/mesosphere/kubeaddons-extrasteps) as a mechanism by which we can generate strong persistent passwords for services pre-startup, would that be appropriate here as well?